src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -180,6 +217,16 @@ def description_wrapper(description)
     end
   end
 
+  def webui2_description_wrapper(description)
```

Comment:
Same thing here, a partial would be nice or even directly in the view.

Reply:
I will move it to a partial